README.md

```diff
@@ -197,6 +197,24 @@ Key Storage | 60405 | Exception while storing/loading key pairs
 Initialization | 60406 | Exception while initializing an object
 Authentication Token | 60407 | Exception while generating token
 
+### Getting server error
+You can control custom server errors listed here (https://www.twilio.com/docs/api/errors) by following the next example:
```

Comment:
Could you change `custom server errors` to `Verify API error codes`?

Reply:
Done!

https://github.com/twilio/twilio-verify-ios/blob/cmejia/22264/error/README.md